src/libraries/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -485,6 +485,19 @@ public bool IsIPv6Teredo
             }
         }
 
+        /// <devdoc>
+        ///   <para>
+        ///     Determines if an address is an IPv6 Unique Local address
+        ///   </para>
+        /// </devdoc>
+        public bool IsIPv6UniqueLocal
+        {
+            get
+            {
+                return IsIPv6 && ((_numbers[0] & 0xFE00) == 0xFC00);
```

Comment:
Might need to ! away nullability on _numbers

Reply:
```suggestion
                return IsIPv6 && ((_numbers[0]! & 0xFE00) == 0xFC00);
```

Like this, sure? Got it!